go-controller/pkg/node/node_ip_handler_linux.go

```diff
@@ -128,6 +126,21 @@ func (c *addressManager) isValidNodeIP(addr net.IP) bool {
 	return true
 }
 
+// sets address annotations on the node
+func (c *addressManager) setAnnotations() error {
+	node, err := c.kube.GetNode(c.nodeName)
+	if err != nil {
+		return fmt.Errorf("unable to retrieve node %s: %v", c.nodeName, err)
+	}
+	nodeAnnotator := kube.NewNodeAnnotator(c.kube, node)
```

Comment:
NIT: this can be moved to line 140, closer to Run()

Reply:
Not sure if needed, there is a log for this [here](https://github.com/ovn-org/ovn-kubernetes/blob/464f5a0723b7ec13d0020c735dcb96fa1c1c7df1/go-controller/pkg/kube/kube.go#L102)

Reply:
`SetNodeHostAddresses` uses the `nodeAnnotator`

Reply:
> `SetNodeHostAddresses` uses the `nodeAnnotator`

duh, I missed that.